[2024-08-18T16:21:13.373+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.scale_data manual__2024-08-18T16:20:56.467671+00:00 [queued]>
[2024-08-18T16:21:13.377+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.scale_data manual__2024-08-18T16:20:56.467671+00:00 [queued]>
[2024-08-18T16:21:13.378+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-08-18T16:21:13.385+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): scale_data> on 2024-08-18 16:20:56.467671+00:00
[2024-08-18T16:21:13.391+0000] {standard_task_runner.py:60} INFO - Started process 16798 to run task
[2024-08-18T16:21:13.393+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl', 'scale_data', 'manual__2024-08-18T16:20:56.467671+00:00', '--job-id', '124', '--raw', '--subdir', 'DAGS_FOLDER/etl_process.py', '--cfg-path', '/tmp/tmpimq99n74']
[2024-08-18T16:21:13.394+0000] {standard_task_runner.py:88} INFO - Job 124: Subtask scale_data
[2024-08-18T16:21:13.424+0000] {task_command.py:423} INFO - Running <TaskInstance: etl.scale_data manual__2024-08-18T16:20:56.467671+00:00 [running]> on host f0f27744fe62
[2024-08-18T16:21:13.476+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='scale_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-18T16:20:56.467671+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-18T16:20:56.467671+00:00'
[2024-08-18T16:21:13.477+0000] {etl_process.py:286} INFO - Leyendo Datos
[2024-08-18T16:21:13.488+0000] {etl_process.py:293} INFO - Datos Leidos
[2024-08-18T16:21:13.494+0000] {etl_process.py:315} INFO - X_train_scaled guardado en PostgreSQL en la tabla: X_train_scaled
[2024-08-18T16:21:13.494+0000] {etl_process.py:316} INFO - X_train_scaled en XCOM con key: X_train_scaled
[2024-08-18T16:21:13.923+0000] {etl_process.py:320} INFO - X_test_scaled guardado en PostgreSQL en la tabla: X_test_scaled
[2024-08-18T16:21:13.924+0000] {etl_process.py:321} INFO - X_test_scaled en XCOM con key: X_test_scaled
[2024-08-18T16:21:14.015+0000] {etl_process.py:325} INFO - y_train_scaled guardado en PostgreSQL en la tabla: y_train_scaled
[2024-08-18T16:21:14.016+0000] {etl_process.py:326} INFO - y_train_scaled en XCOM con key: y_train_scaled
[2024-08-18T16:21:14.099+0000] {etl_process.py:330} INFO - y_test_scaled guardado en PostgreSQL en la tabla: y_test_scaled
[2024-08-18T16:21:14.100+0000] {etl_process.py:331} INFO - y_test_scaled en XCOM con key: y_test_scaled
[2024-08-18T16:21:14.196+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-08-18T16:21:14.204+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl, task_id=scale_data, execution_date=20240818T162056, start_date=20240818T162113, end_date=20240818T162114
[2024-08-18T16:21:14.249+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-08-18T16:21:14.261+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
